concepts/05 UI Components/DataGrid/99 How To/Customize Header Filter Data Source/20 Map Data Source Fields.md

```diff
@@ -31,7 +31,9 @@ In the following code, the `categoryName` and `categoryId` fields are mapped to
                                 categoryId: item.categoryId,
                                 categoryCode: item.categoryCode
                             }
-                        }
+                        },
+                        // Turn pagination off if header filter items are sorted incorrectly:
```

Comment:
In the original doc issue T818801 postProcess is used to sort items. In this case paginate: false is required. But in all other cases paginate: false do not change sorting. I think we should not add this very specific comment here. Or add some information to DataSource.postProcess topic.